manage.py

```diff
@@ -102,7 +102,7 @@ def _add_output_subparser(subparsers):
     # Output service options
     output_parser.add_argument(
         '--service',
-        choices=['aws-lambda', 'aws-s3', 'pagerduty', 'phantom', 'slack'],
+        choices=['aws-lambda', 'aws-s3', 'pagerduty', 'pagerduty-v2', 'phantom', 'slack'],
```

Comment:
I don't think your implementation of this uses a hyphen does it? Shouldn't this just be `pagerdutyv2`. See: [here](https://github.com/airbnb/streamalert/blob/master/stream_alert/alert_processor/outputs.py#L126)

Reply:
Not a typo, but trying to change that to use the hyphen, and also see @jacknagz comment above.